tests/cli/workflow/nf_analysis/test_cli_config_case.py

```diff
@@ -55,6 +57,11 @@ def test_config_with_missing_case(
     caplog.set_level(logging.ERROR)
     context: CGConfig = request.getfixturevalue(f"{workflow}_context")
 
+    # In the RAREDISEASE case, we need to mock lims fetching of the target bed file
+    if workflow == Workflow.RAREDISEASE:
+        mocker.patch.object(RarediseaseAnalysisAPI, "get_target_bed_from_lims")
+        RarediseaseAnalysisAPI.get_target_bed_from_lims.return_value = "some_target_bed_file"
```

Comment:
```suggestion
        mocker.patch.object(AnalysisAPI, "get_target_bed_from_lims", return_value=DEFAULT_CAPTURE_KIT
```

Reply:
Also, would it make sense to move it to the `raredisease_context` fixture so we avoid repetition?